PlantPal/PlantPal/View/ReusableView/PalText.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PalText.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 20/08/2024.
+//
+
+import SwiftUI
+
+// MARK: - PalText
+
+struct PalText: View {
+    private let text: String
+    private let fontType: FontType
+    private let size: CGFloat
+
+    init(text: String, fontType: FontType, size: CGFloat) {
+        self.text = text
+        self.fontType = fontType
+        self.size = size
+    }
+    
+    var body: some View {
+        Text(text)
+            .font(.custom(fontType.fontName, size: size))
+    }
+}
+
+enum FontType {
+    case regular
+    case bold
+    case italic
+    case light
+    
+    var fontName: String {
+        switch self {
+        case .regular:
+            return "Merriweather-Regular"
+        case .bold:
+            return "Merriweather-Bold"
+        case .italic:
+            return "Merriweather-Italic"
+        case .light:
+            return "Merriweather-Light"
+        }
+    }
+}
```

Comment:
Nie potrzebujemy tutaj tego property. 
```suggestion
enum FontType: String {
    case regular = "Merriweather-Regular"
    case bold = "Merriweather-Bold"
    case italic = "Merriweather-Italic"
    case light = "Merriweather-Light"
}
```

Reply:
Imo ten FontType powinien być wewnątrz PalText.